sio-digital-team/active-safe
# Wordpress wp-content files

https://trello.com/c/tlzrHneT


**Changes:**

- [x] Move file into theme folder of wordpress
- [x] Include plugins, and uploads folder
- [x] Split index files to snippets

**How to test:**
1) Install Docker https://docs.docker.com/docker-for-mac/install/
1) Clone the repo into a new folder you wish to run docker
1) Run `docker-compose up -d` in the root project file. This will provide a fresh install of wordpress
1) Go into the theme by running `cd wp-content/themes/activeandsafe`
1) Run `yarn` within the "activeandsafe" theme in order to grab all the dependencies
1) Run `yarn watch` to compile all the assets for the site to use
1) Visit site to check if its working at http://localhost:8000
1) Go through the wordpress installation guide that shows up
1) Log into the dashboard using the login credentials you created for yourself
1) Activate the "Active and Safe Central" theme
